src/utils/validateScenarios.test.tsx

```diff
@@ -28,25 +28,8 @@ function fail(entry: FileEntry, rx: RegExp | string) {
   expect(() => validateScenarios([entry])).toThrowError(rx);
 }
 
-const baseScenario = {
-  id: "scn-1",
-  name: "Sample",
-  description: "desc",
-  pathwayType: "Normative",
-  modelYearStart: 2000,
-  modelYearEnd: 2050,
-  publicationYear: 2020,
-  regions: ["Global"],
-  sectors: [{ name: "Power", technologies: ["Solar"] }],
-  publisher: "RMI",
-  overview: "overview",
-  expertRecommendation: "rec",
-  dataSource: {
-    description: "src",
-    url: "https://example.com/file.csv",
-    downloadAvailable: true,
-  },
-};
+import rawScenarioArray from "../../testdata/valid/scenarios_metadata_standard.json" assert { type: "json" };
```

Comment:
NB:
Ah, I see what you mean now. It does feel a tiny bit strange to have code in `src` depend on anything outside of `src`. 

I don't know what the common patterns are in React for unit testing, but I wonder if at some point, we want to just extract all `*.test.tsx` files into a `tests` dir outside of src. Unless there is a benefit to bundling the tests in with the source code that I'm not thinking of

Reply:
Anyway, probably a larger architecture discussion, and don't want that discussion to block this PR

Reply:
agreed. I don't think it's that big an issue, but it would be good to talk with someone who is JS-native, rather than "a bunch of R devs who picked up JS"